### ISSUE TITLE
Rabbit always reports sockets_used as 0

### ISSUE THREAD

**Issue by essen** [rabbitmq/rabbitmq-server#446]
Due to the switch to Ranch and my removing of important code the sockets_used value always returns 0 and causes a field in the management UI to be missing. Offending commit: 93f17772a0767194c08129956ce52399d56a78a1

**Comment by essen** [comment]
It appears I also removed the "tune_buffer_size" functionality.

**Comment by essen** [comment]
I spent a lot of time on tune_buffer_size, perhaps more than I should have, but now I understand why it was there much better. The best solution would be to have a callback in Ranch to perform this operation right after accepting (like it was before the switch to Ranch). I have opened a ticket for Ranch at https://github.com/ninenines/ranch/issues/118 but I don't expect it to be done before the end of the week.

For the time being I will do a solution that is not the best but will work. I also identified a missing ranch:accept_ack call that I will fix in the same branch on a separate commit.
